changelog/README.md

```diff
@@ -21,6 +21,30 @@ under the License.
 
 <!-- Note: contrary to 3.x, insert new entries *first* in their section -->
 
+### 4.19.0
+
+- [bug] JAVA-3055: Prevent PreparedStatement cache to be polluted if a request is cancelled.
+- [bug] JAVA-3168: Copy node info for contact points on initial node refresh only from first match by endpoint
+- [improvement] JAVA-3143: Extend driver vector support to arbitrary subtypes and fix handling of variable length types (OSS C* 5.0)
+- [improvement] CASSJAVA-53: Update Guava version used in cassandra-java-driver
+- [improvement] JAVA-3118: Add support for vector data type in Schema Builder, QueryBuilder
+- [bug] CASSJAVA-55: Remove setting "Host" header for metadata requests
+- [bug] JAVA-3057: Allow decoding a UDT that has more fields than expected
+- [improvement] CASSJAVA-52: Bring java-driver-shaded-guava into the repo as a submodule
+- [bug] CASSJAVA-2: TableMetadata#describe produces invalid CQL when a type of a column is a vector
+- [bug] JAVA-3051: Memory leak
```

Comment:
I'm kinda thinking we might want some additional detail on this one; I don't want people seeing this and thinking there was some kind of broad-based memory leak in the driver

Reply:
Very much agreed @tolbertam.  I'm gonna take a pass at fixing the verbiage here.

FTR this specific issue isn't explicitly connected to AWS Keyspaces.  Apps using Keyspaces were more likely to see it because of the reconnect loop discussed elsewhere in the ticket but in theory any driver app could see this issue.